### PR TITLE
Add missing docker file for ubuntu 1804

### PR DESCRIPTION
**Pull request by Marenz**

### CODE REVIEW

File: .circleci/docker/Dockerfile.1804 [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+RUN set -ex; \
+	apt-get update; \
+	apt-get install -qqy --no-install-recommends \
+		build-essential \
+		software-properties-common \
+		ninja-build clang++-8 \
+		libboost-regex-dev libboost-filesystem-dev libboost-test-dev libboost-system-dev \
+		libboost-program-options-dev \
+		libjsoncpp-dev \
+		llvm-8-dev libcvc4-dev \
```
Comment (ekpyron) [inline .circleci/docker/Dockerfile.1804]: We can remove ``libcvc4-dev`` right away and instead add ``libz3-static-dev`` here and on the top before the ``apt-get update`` the same as in the 1904 docker file for adding the deps PPA:
```
    dist=$(grep DISTRIB_CODENAME /etc/lsb-release | cut -d= -f2); \
    echo "deb http://ppa.launchpad.net/ethereum/cpp-build-deps/ubuntu $dist main" >> /etc/apt/sources.list ; \
    apt-key adv --keyserver keyserver.ubuntu.com --recv-keys 1c52189c923f6ca9 ; \
```

**Comment by leonardoalt** [comment]
So this still uses Aleth instead of evmone?

**Comment by ekpyron** [comment]
If it's still aleth instead of evmone, we have to change that as well.

**Comment by ekpyron** [comment]
(I mean we don't have to, because we don't have a test run, but we should do it anyways and maybe even add a test run for it - we release for this ubuntu version after all)

**Comment by christianparpart** [comment]
* PR tanken over
* rebased, hence removed Aleth (using EVMONE)
* adapted Dockerfile.ubuntu1804 to match Ubuntu 19.04's dockerfile.

**Comment by Marenz** [comment]
LGTM

File: .circleci/docker/Dockerfile.ubuntu1804 [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+		llvm-8-dev libz3-static-dev \
+		; \
+	apt-get install -qy python-pip python-sphinx; \
+	update-alternatives --install /usr/bin/llvm-symbolizer llvm-symbolizer /usr/bin/llvm-symbolizer-8 1; \
+	pip install codecov; \
+	rm -rf /var/lib/apt/lists/*
+
+#  Cmake
+RUN set -ex; \
+	wget -q -O /tmp/cmake.sh https://cmake.org/files/v3.13/cmake-3.13.0-Linux-x86_64.sh; \
```
Comment (ekpyron) [inline .circleci/docker/Dockerfile.ubuntu1804]: It'd still be nice to get rid of this and either download a binary evmone release or patch evmone's CMakeLists as you suggested...

**Review by ekpyron — DISMISSED** [review]

**Comment by ekpyron** [comment]
Needs to be rebased (which should fix the ossfuzz build failure).

**Review by ekpyron — APPROVED** [review]
I actually thought this was merged already for quite a while :-).

**Comment by ekpyron** [comment]
@chriseth Can you force-merge this? For some reason github still considers the b_ubu_cxx17 run required here...

**Pull request merged**